src/util/apiCalls.test.js

```diff
@@ -333,27 +333,6 @@ describe('postPalette', () => {
       });
     });
   
-    it('should be passed the correct url', () => {
-       const expected = [ `https://mysterious-dusk-17585.herokuapp.com/api/v1/projects/${projectId}/palettes`,{
-        method: 'POST',
-           body: JSON.stringify({
-            'name': paletteName,
-            'color1': color1,
-            'color2': color2,
-            'color3': color3,
-            'color4': color4,
-            'color5': color5
-          }),
-        headers: {
-          'Content-Type': 'application/json'
-        }
-      }];
-        
-      postPalette(paletteName, color1, color2, color3, color4, color5, projectId)
-  
-      expect(window.fetch).toHaveBeenCalledWith(...expected);
-    })
-  
```

Comment:
Good call on deleting this test.  Based on what I'm hearing from other people, we don't need to test to see if the request has been called with the url and should focus more on the response status and body.